src/OpenSheetMusicDisplay/OpenSheetMusicDisplay.ts

```diff
@@ -563,5 +568,35 @@ export class OpenSheetMusicDisplay {
     public get Version(): string {
         return this.version;
     }
+
+    /**
+     * set player
```

Comment:
Comment: set
function name: create

It uses a singleton pattern so injection would be ok if you were going for it but only a callback is passable without description.